src/routes/(app)/checkout/+page.server.ts

```diff
@@ -14,13 +14,42 @@ import type { JsonObject } from 'type-fest';
 import { omit, set } from 'lodash-es';
 import { rateLimit } from '$lib/server/rateLimit.js';
 import { cmsFromContent } from '$lib/server/cms';
+import { computePriceInfo } from '$lib/types/Cart';
+import { UNDERLYING_CURRENCY } from '$lib/types/Currency';
+import { toCurrency } from '$lib/utils/toCurrency';
 
 export async function load({ parent, locals }) {
 	const parentData = await parent();
 
 	if (parentData.cart) {
 		try {
 			await checkCartItems(parentData.cart, { user: userIdentifier(locals) });
+			const priceInfoInitial = computePriceInfo(parentData.cart, {
+				bebopCountry: runtimeConfig.vatCountry,
+				vatSingleCountry: runtimeConfig.vatSingleCountry,
+				vatNullOutsideSellerCountry: runtimeConfig.vatNullOutsideSellerCountry,
+				vatExempted: runtimeConfig.vatExempted,
+				userCountry: locals.countryCode,
+				deliveryFees: {
+					amount: 0,
+					currency: UNDERLYING_CURRENCY
+				},
+				vatProfiles: parentData.vatProfiles
+			});
+
+			const isDigital = parentData.cart?.every((item) => !item.product.shipping);
+			const physicalCartCanBeOrdered =
+				!isDigital &&
+				!!runtimeConfig.physicalCartMinAmount &&
+				priceInfoInitial.partialPriceWithVat >=
+					toCurrency(
+						priceInfoInitial.currency,
+						runtimeConfig.physicalCartMinAmount,
+						runtimeConfig.mainCurrency
+					);
+			if (!physicalCartCanBeOrdered) {
+				throw error(403, `Can't order a cart with amount < ${runtimeConfig.physicalCartMinAmount}`);
+			}
```

Comment:
I think this error should be shown on the checkout page itself (like the cart error message) instead of crashing the page